package.json

```diff
@@ -9,9 +9,10 @@
     "compile": "webpack",
     "karma": "karma start karma.config.js",
     "lint": "eslint '**/*.js'",
+    "lint-fix": "eslint '**/*.js' --fix",
     "prepublish": "yarn compile",
     "start": "webpack-dev-server",
-    "test": "yarn ci:lint && yarn ci:test",
+    "test": "yarn ci:test && yarn ci:lint",
```

Comment:
No. Because lint fix will not fix everything. Might as well get into the habit of writing lint-free code before running specs.